ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -2,57 +2,97 @@
 
 This folder contains the files to configure IPsec encryption for external traffic
 
-* Enable IPsec encryption yaml
-* IPsec endpoint configuration with NMState yml
-* build.sh script that creates a MachineConfig to import external certs into IPsec NSS and apply the IPsec configuration
-
-Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
+Also available in OpenShift docs: [IPsec encryption for external traffic](https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn)
 
 ### Prerequisites
 
 * `butane` utility installed, minimal version 0.20.0.
+* An IPsec endpoint certificate for each node in PKCS#12 format named the with the nodename and a CA cert in PEM format.
+Optionally, a single certificate for all the nodes (the nodes' IDs (IP, DNS, etc) should be set in the certificate SAN extension).
+* Include `enable-ipsec.yaml` file in one of the additional install-time manifests directories defined in the `extraManifests.searchPaths` field in the SiteConfig file
+More info about the `extraManifests.searchPaths` mechanism can be found in the [GitOps ZTP README](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/README.md)
+
+Below are two options to configure IPsec encryption for external traffic.
+
+### IPsec configuration with NMState Operator
+
+#### Prerequisites
+
+* NMState Operator installed on the cluster.
```

Comment:
shouldnt we include the subscription for it?
not an expert here, but I believe that's how it's done in other cases,
for example
https://github.com/openshift-kni/cnf-features-deploy/blob/e31eb6cd0b0b2f09472b5de086302656abdb2a48/ztp/source-crs/PtpSubscription.yaml#L5
not sure how to do an optional one though..

Reply:
To make it optional, Add the source-crs for it, but when you add it to the example PGT/PolicyGenerator, comment it out (like [this](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml#L59-L67))